### DISCUSSION TITLE
如何搭配Adguard Home使用？

### DISCUSSION THREAD

**Discussion by bolisci** [Aethersailor/Custom_OpenClash_Rules#16] [General]
感谢repo作者细致的教程！我看到教程有通过开发者选项去广告的方法。但我仍想请教一下，有没有办法搭配Adguard Home使用？用它我可以对某些设备进行限制（例如儿童设备），也可以方便地增减屏蔽或放行的域名，还可以查看各设备的DNS请求记录。谢谢！

**Comment by Aethersailor** [comment]
因为 openclash 对 53 端口重定向的原因，本仓库的教程设置不适合在同一系统下搭配 adguardhome 使用
你可以百度一下其他的设置方法，网上有很多 openclash 和 adguardhome 搭配方案
